source/elements/oneDPL/source/parallel_api/buffer_wrappers.rst

```diff
@@ -0,0 +1,73 @@
+.. SPDX-FileCopyrightText: 2019-2022 Intel Corporation
+.. SPDX-FileCopyrightText: Contributors to the oneAPI Specification project.
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+Buffer Wrappers
+---------------
+
+.. code:: cpp
+
+  // Defined in <oneapi/dpl/iterator>
+
+  namespace oneapi {
+    namespace dpl {
+
+      template < typename T, typename AllocatorT, typename TagT >
+      /*unspecified*/ begin( sycl::buffer<T, /*dim=*/1, AllocatorT> buf,
+                             TagT tag = sycl::read_write );
+
+      template < typename T, typename AllocatorT, typename TagT >
+      /*unspecified*/ begin( sycl::buffer<T, /*dim=*/1, AllocatorT> buf,
+                             TagT tag, sycl::property::no_init );
+
+      template < typename T, typename AllocatorT >
+      /*unspecified*/ begin( sycl::buffer<T, /*dim=*/1, AllocatorT> buf,
+                             sycl::property::no_init );
+
+
+      template < typename T, typename AllocatorT, typename TagT >
+      /*unspecified*/ end( sycl::buffer<T, /*dim=*/1, AllocatorT> buf,
+                           TagT tag = sycl::read_write );
+
+      template < typename T, typename AllocatorT, typename TagT >
+      /*unspecified*/ end( sycl::buffer<T, /*dim=*/1, AllocatorT> buf,
+                           TagT tag, sycl::property::no_init );
+
+      template < typename T, typename AllocatorT >
+      /*unspecified*/ end( sycl::buffer<T, /*dim=*/1, AllocatorT> buf,
+                           sycl::property::no_init );
+
+    }
+  }
+
+``oneapi::dpl::begin`` and ``oneapi::dpl::end`` are helper functions
+for passing SYCL buffers to oneDPL algorithms.
```

Comment:
Since we have moved this away from the other page, do we need to make this reference to `SYCL` a link?

Reply:
Done.